packages/components/src/components/numpad/numpad.tsx

```diff
@@ -12,20 +12,20 @@ type TNumpad = {
     is_regular?: boolean;
     is_currency?: boolean;
     is_submit_disabled?: boolean;
-    label: string;
+    label?: string;
     reset_press_interval: number;
     reset_value: string;
-    max: number;
-    min: number;
+    max?: number;
+    min?: number;
     pip_size: number;
     onSubmit: (param: number | string) => void;
-    v: string;
+    v?: string;
     render?: (props: { value: string; className: string }) => React.ReactNode;
     submit_label: string;
-    value: string;
-    format: (v: string) => number;
+    value: string | number;
+    format?: (v: string) => number | string;
     onValueChange: (val: number | string) => void;
-    onValidate: (default_value: number | string) => string | undefined;
+    onValidate: (default_value: number | string) => boolean | 'error';
```

Comment:
what is this string 'error' is referring to ?

Reply:
https://github.com/binary-com/deriv-app/pull/9455/commits/bae55f77afee63ad9982261d9fc371379252c44d#diff-e5010e7cb32477426b582230ad84afaafd16f4f5e29a1de1b3447772f9a224d3

Reply:
it was me 😹. but looks correct lol